pkg/acs/calacs/acsrej/acsrej_loop.c

```diff
@@ -332,13 +390,13 @@ int acsrej_loop (IODescPtr ipsci[], IODescPtr ipdq[],
 
     /* Assumption: ALL images have the same noise/gain values */
     for (k = 0; k < NAMPS; k++) {
-        gain2[k] = 0.;
+        gain2[k] = 0.;  /* e/DN, copied from gain below */
         noise2[k] = 0.;
```

Comment:
Note to self: If there were time, many variable names need to be clarified.  In particular, gain2 is not a squared value (it seems), yet is named as if it were.  Readability of code is reduced.  

Note this comment is based upon the original code and not the changes made for the new ACSREJ algorithm.